app/lib/user_action_events/yaml_validator.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module UserActionEvents
```

Comment:
It's a bit confusing and potentially problematic to have `UserActionEvents` module and `UserActionEvent` class.